contrib/google.golang.org/grpc/appsec_test.go

```diff
@@ -24,26 +28,26 @@ import (
 )
 
 func TestAppSec(t *testing.T) {
-	appsec.Start()
+	appsec.Start(appsecConfig.WithWAFTimeout(time.Hour /* functionally unlimited */))
```

Comment:
@RomainMuller If `WithWAFTimeout` is only for this, could we just set the environment variable?

```suggestion
	t.SetEnv("DD_APPSEC_WAF_TIMEOUT", "1h")
	appsec.Start()
```